front-end/src/components/player-page/stats/stats-generators/AverageVisionScoreStatGenerator.ts

```diff
@@ -2,21 +2,15 @@ import {BaseStatGenerator} from "./BaseStatsGenerator";
 import PlayerStatModel from "../../../../../../Common/models/playerstat.model";
 
 export class AverageVisionScoreStatGenerator extends BaseStatGenerator {
-    getStatValue(playerStatsModels: PlayerStatModel[]): string {
-        let vs = 0
-        let games = 0;
-        for (let playerStatsModel of playerStatsModels) {
-            vs += playerStatsModel.visionScore;
-            games += playerStatsModel.games;
-        }
-        return `${this.formatNumber(vs/games)}` ;
-    }
-
     getStatTitle(): string {
         return "AVG VS"
     }
 
     getToolTip(): string {
         return "Average Vision Score Per Game";
     }
+
+    getStatValue(playerStatsModel: PlayerStatModel): number {
```

Comment:
Add rounding

Reply:
We dont want rounding here, this function is to get the raw stat value. Rounding should only happen when the value gets converted to a string. The raw value is useful for leaderboard rankings where you care about decimal precision.